Agreements/External/18_RR-E.txt

```diff
@@ -0,0 +1,28 @@
+;requires an LoA change
```

Comment:
Still discussing with Pierre what we're going to do on this one. But this will need to be from Sector 18, rather than Sector 1.

Reply:
When you decide let us know how to implement

Reply:
Max level 290 is now agreed

Reply:
rgr - done